src/api/spec/models/project_spec.rb

```diff
@@ -469,4 +469,50 @@
       it { expect(Project.deleted?(project.name)).to be_truthy }
     end
   end
+
+  describe '#render_relationships' do
+    let!(:project) { create(:project) }
+    let!(:group) { create(:group) }
+    let!(:role) { Role.find_by_title('bugowner') }
```

Comment:
[nitpick] the bang operator is not necessary for project, group and role.

Reply:
I've always just added the bang ! to every let that I know needs to be eagerly executed since it seems easier then going through the code and figuring out which lets need ! and which don't.